src/ds/d400/d400-private.h

```diff
@@ -139,6 +142,7 @@ namespace librealsense
             {RS_D400_USB2_RECOVERY_PID, "5.8.15.0"},
             {RS400_IMU_PID, "5.8.15.0"},
             {RS420_PID, "5.8.15.0"},
+            {RS421_PID, "5.16.6.0"}, // TODO - D421, update
```

Comment:
We need to decide what we do here.
1 option is do perform fw update only on IPDev and set this as 5.17.0.0
2. Will this block us updating dev versions?

Let's speak and decide together how to handle it

Reply:
This PR does not include the new requirement to block D421 FW update completely, currently only removing from bundled FW checks.

OK, we'll talk and decide how to implement